shared/build.gradle.kts

```diff
@@ -28,8 +30,13 @@ kotlin {
         framework {
```

Comment:
Do we need to update this `ios.deploymentTarget = "14.1"` to match with our `Podfile`? 🤔

Reply:
The podfile of the iOS template has a [bug](https://github.com/nimblehq/ios-templates/issues/509) here. I will create a task to update the minimum iOS version of the development pod with Mustache.

Reply:
So we will have to wait for the ios-template to fix it first? 🙏

Reply:
solved in [2b6549e](https://github.com/nimblehq/kmm-templates/pull/19/commits/2b6549e43b56f9a5f62ae0dec7b33a4c59691359)